server/index.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express')
+const app = express()
+const bodyParser = require('body-parser')
+const respondStatic = express.static('./server/public')
+const podcasts = require('./seed.js')
+const levelup = require('level')
+const db = levelup('../mydb')
+
+app.use(respondStatic)
+
+db.put('podcasts', JSON.stringify(podcasts), err => {
+  if (err) return console.log('Opps!', err)
+})
```

Comment:
you want to extract this initial `db.put` into something like `seed.js`, and then have an npm script `seed` that will run that file.